eland/utils.py

```diff
@@ -185,16 +189,19 @@ def pandas_to_eland(
     actions = []
     n = 0
     for row in pd_df.iterrows():
-        # Use index as _id
-        id = row[0]
-
         if es_dropna:
             values = row[1].dropna().to_dict()
         else:
             values = row[1].to_dict()
 
-        # Use integer as id field for repeatable results
-        action = {"_index": es_dest_index, "_source": values, "_id": str(id)}
+        if ignore_index:
```

Comment:
Maybe we should change the name of this parameter to be more verbose but also clear on what "index" is being ignored since the concept appears in both pandas and ES.

Maybe `ignore_pandas_index` or even flip the default and use `use_pandas_index_as_id=True`?